dist/src/tests/BackendInt.test.js

```diff
@@ -0,0 +1,56 @@
+// Import the JavaTokenizer class from the lexer.ts file
+var JavaTokenizer = require('../logic/lexers').JavaTokenizer;
+var javaCode = "\npublic class HelloWorld {\n  public static void main(String[] args) {\n    System.out.println(\"Hello, 5th World!\");\n    // I am a comment\n    int num = 50/2;\n  }\n  /*\n  I am multiple lines >2\n  but also a comment\n  */\n}\n";
+// Define the expected tokens for the given Java code
+var expectedTokens = [
```

Comment:
This test seems to be failing, probably because of the recent changes in the lexer. Should be a pretty easy fix, I think some of the token types were just renamed